index.html

```diff
@@ -359,12 +359,44 @@ <h3>
       additional members to control audio packetization.
     </p>
     <pre class="idl">partial dictionary RTCRtpEncodingParameters {
-  unsigned long ptime;
-  boolean adaptivePtime = false;
+    RTCResolutionRestriction scaleResolutionDownTo;
+    unsigned long            ptime;
+    boolean                  adaptivePtime = false;
 };</pre>
     <section id="rtcrtpencodingparameters-attributes">
       <h2>Dictionary {{RTCRtpEncodingParameters}} Members</h2>
       <dl data-link-for="RTCRtpEncodingParameters" data-dfn-for="RTCRtpEncodingParameters" class="dictionary-members">
+        <dt>
+          <dfn data-idl>scaleResolutionDownTo</dfn> of type <span class="idlMemberType">{{RTCResolutionRestriction}}</span>
+        </dt>
+        <dd>
+          <p>The maximum dimensions at which to restrict this encoding.</p>
+          <p>When {{scaleResolutionDownTo}} is specified, the
+          {{RTCRtpEncodingParameters/scaleResolutionDownBy}} value MUST
+          be ignored. Instead, frames are sent according to the
+          specified resolution restrictions: frames MUST NOT be upscaled. In simulcast, not
+          upscaling can result in multiple same-sized encodings if the size of
+          the frame is smaller than {{scaleResolutionDownTo}}; an encoding MUST
+          NOT be sent if {{scaleResolutionDownTo}} is larger than the frame and
+          there exists another {{RTCRtpEncodingParameters/active}} encoding with
+          a smaller {{scaleResolutionDownTo}} density resulting in the same
```

Comment:
Can we defer this part about simulcast to another issue? This is supposed to be a max value which implies lower resolutions should be sent, but here it says the opposite. Might need a bit more discussion

Reply:
Yes, I filed #222